.yarn/versions/baeeda33.yml

```diff
@@ -0,0 +1,2 @@
+releases:
+  "@moonrepo/cli": patch
```

Comment:
need to bump all the core packages too, try `yarn version:bump:bin patch`

Reply:
👍